spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/resources/zookeeper-deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kafka
+    component: zookeeper
+  name: zookeeper
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kafka
+      component: zookeeper
+  template:
+    metadata:
+      labels:
+        app: kafka
+        component: zookeeper
+    spec:
+      containers:
+      - name: zookeeper
+        image: wurstmeister/zookeeper:latest
```

Comment:
Same here, I think we should pick a specific version

Reply:
Thanks. Seems like we will need to use a different image since wurstmeister does not have tags that are any recent. pushed an update to use the official zookeeper image instead.